src/pages/Login/Login.js

```diff
@@ -1,7 +1,20 @@
 import React from 'react';
+import './Login.scss';
 
 const Login = () => {
-  return <div>Login</div>;
+  return (
+    <main className="login-container">
```

Comment:
```suggestion
    <main className="login">
```

className은 컴포넌트의 이름과 동일하게 작성해주세요!